docs/common/registration.md

```diff
@@ -102,8 +106,8 @@ To register a new resource, follow the instructions below:
 
 To modify an existing resource, follow these instructions:
 
-1. Find the resource that you would like to modify by [searching GitHub](#searching-for-resources) and opening the link
-   to the YAML file.
+1. Find the resource that you would like to modify by [searching GitHub](#searching-for-resources) and open the link to
```

Comment:
Just a thought, I feel like it might read more easily with a comma before "and open"